packages/storybook/stories/QuickStarts/QuickstartComposite.stories.mdx

```diff
@@ -44,8 +44,11 @@ Access Tokens can be generated using this [Azure Portal quickstart](https://docs
 
 ## Setting Up
 
-UI Library requires a React environment to be setup. Next we will do that. If you are creating a new app or are
-having issues since your app is running react 18 or higher visit our [setup page](./?path=/docs/setup-communication-react--page).
+UI Library requires a React environment to be setup. Follow the [setup page](./?path=/docs/setup-communication-react--page) to get started
+with your new app.
+
+If you are on react 18 or higher and are experiencing issues with starting your app,
```

Comment:
nit: 
If you are on react 18 or higher and experiencing issues with starting your app